docs/sql-ref-syntax-qry-select-join.md

```diff
@@ -18,5 +18,236 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+A SQL join is used to combine rows from two relations based on join criteria. The following section describes the overall join syntax and the sub-sections cover different types of joins along with examples.
+
+### Syntax
+{% highlight sql %}
+
+relation { [ join_type ] JOIN relation [ join_criteria ] | NATURAL join_type JOIN relation }
+
```

Comment:
Not need to introduce NATURAL JOIN?

Reply:
I am OK either way. 
@maropu WDYT?

Reply:
Yea, better to document `NATURAL JOIN`, too, as the Pg doc does so: https://www.postgresql.org/docs/current/sql-select.html